vendor/engines/split_accounts/spec/services/order_detail_splitter_spec.rb

```diff
@@ -13,8 +12,10 @@
     end
   end
 
+  let(:order) { build_stubbed(:order) }
+
   let(:order_detail) do
-    build_stubbed(:order_detail).tap do |order_detail|
+    build_stubbed(:order_detail, order: order).tap do |order_detail|
```

Comment:
I try to use `stubbed_build` instead of `build` if possible so I don't end up creating database entries for associations (something factory girl likes to do).